doc/source/user_guide/troubleshoot.rst

```diff
@@ -148,6 +150,7 @@ If this command doesn't launch MAPDL, look at the command output:
     terminated Ansys run. To disable this check, set the ANSYS_LOCK environment
     variable to OFF.
```

Comment:
```suggestion
    terminated Ansys run. To disable this check, set the ``ANSYS_LOCK`` environment
    variable to ``OFF``.
```

Reply:
Because this is output "as-it-is" from MAPDL, I would leave it as it is.